cmd/query/app/http_handler.go

```diff
@@ -132,11 +132,9 @@ func (aH *APIHandler) handleFunc(
 ) *mux.Route {
 	route := aH.formatRoute(routeFmt, args...)
 	var handler http.Handler = http.HandlerFunc(f)
-	traceMiddleware := otelhttp.NewHandler(
-		otelhttp.WithRouteTag(route, traceResponseHandler(handler)),
-		route,
-		otelhttp.WithTracerProvider(aH.tracer))
-	return router.HandleFunc(route, traceMiddleware.ServeHTTP)
+	handler = traceResponseHandler(handler)
```

Comment:
TODO traceResponseHandler should be moved to initRouter, it can apply to the whole server, not just API handler

Reply:
@yurishkuro do we want to do that in this PR or a different one?

Reply:
probably another, already too many changes here to keep track of

Reply:
https://github.com/jaegertracing/jaeger/issues/6141